pkg/cli/create_helm.go

```diff
@@ -322,9 +322,15 @@ func CreateHelm(ctx context.Context, options *CreateOptions, globalFlags *flags.
 	if err != nil {
 		return err
 	}
+
 	verb := "created"
 	if isVClusterDeployed(release) {
 		verb = "upgraded"
+		currentVClusterConfig, err := getConfigfileFromSecret(ctx, vClusterName, cmd.Namespace)
```

Comment:
The `:=` is shadowing both `currentVClusterConfig` and `err`.  Recommend using `=` instead.

Reply:
Good catch, both of them have been declared already. I'll fix it ;)